### ISSUE TITLE
DeclNode missing within named FunctionExpression

### ISSUE THREAD

**Issue by royvandijk06** [PerimeterX/flast#10]
Managed to find the cause:
Whenever you add an id/name to a FunctionExpression within a FunctionExpression call, it won't detect the `declNode` of any variable in this scope.

For example:
![image](https://user-images.githubusercontent.com/15157079/184509778-187d5136-afbb-43e0-9ff4-45d70a587feb.png)

https://runkit.com/royvandijk06/62f7edc05b1e5b00081e37ea

**Comment by royvandijk06** [comment]
I think the issue may be that u need to check variables from upper or more precise scopes?

**Comment by BenBaryoPX** [comment]
As I mentioned in #8, this is now covered in the scopes overhaul of v1.2.1.
Thank you for reporting this issue and suggesting a solution in the PR.
Please reopen this issue if you find the problem still persists.

**Comment by royvandijk06** [comment]
I didn't feel like starting a new issue for this, I hope you don't mind me repurposing this issue.
So, I've made a tool with this library which I published yesterday.
I would like to invite you and your team to take a look at it and provide some feedback, if you have the time.
I have made a question form for this, which also includes instructions:
https://forms.gle/cHxAbWNreDASUiCx5
Thanks again for this library, you've really helped me with it.
